content/posts/08-meet-APKBUILD.vim.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "APKBUILD.vim: filetype, plugins, syntax and plugins for APKBUILDs on vim"
+date: 2020-10-30T17:39:23-03:00
+draft: false
+---
+
+> Article by editor Leo <thinkabit.ukim@gmail.com>
+
+The Linux distro I contribute to, [Alpine Linux][1], uses build recipes called
+APKBUILDs, they are shell scripts but with context-aware variables that can
```

Comment:
```suggestion
APKBUILDs. They are shell scripts but with context-aware variables that can
```
Otherwise the sentence get really long and hard to follow